dev-tools/dependencies-report

```diff
@@ -41,3 +41,14 @@ go list -m -json all $@ | go run go.elastic.co/go-licence-detector \
 		-noticeTemplate "$SRCPATH/notice/dependencies.csv.tmpl" \
 		-noticeOut "$outfile" \
 		-depsOut ""
+
+
+# Fill-in required values for UBI images
+# Check headers in $SRCPATH/notice/dependencies.csv.tmpl:
+# name,url,version,revision,license
+ubi8url='https://catalog.redhat.com/software/containers/ubi8/ubi-minimal/5c359a62bed8bd75a2c3fba8'
```

Comment:
I see is always using `latest` in https://github.com/elastic/beats/pull/20576/files#diff-4b50598f6acbbdc2d3c22be81ed43eceR487

What do you think to force the tag to a particular version in `dev-tools/packaging/packages.yml`? Then read the tag from the above file to be included in the `ubi8url`?

For instance for the  tag `8.2` the `ubi8url` would be `https://catalog.redhat.com/software/containers/ubi8/ubi-minimal/5c359a62bed8bd75a2c3fba8?tag=8.2`

Reply:
For the certification purpouses it seems to be enough to link to the base URL. Also this way we don't need to update this every time we upgrade the base image to a new minor.